mdpow/top/charmm36Dummy2.ff/forcefield.doc

```diff
@@ -0,0 +1,152 @@
+
+
+CHARMM36 all-atom force field plus dummy2
+
+*******************************************************************************
+*                    CHARMM36 port writted by                                 *
+*                    E. Prabhu Raman, Justin A. Lemkul, Robert Best           *
+*                    and Alexander D. MacKerell, Jr.                          *
+*                    CHARMM force field homepage:                             *
+*                    www.mackerell.umaryland.edu/CHARMM_ff_params.html        *
+*******************************************************************************
```

Comment:
Add a line with your name and state that you added the non-bonded dummy model for ions